upgrading-info/armour-and-weapons.txt

```diff
@@ -75,7 +75,8 @@ _ _
 <:noxscythe:513190159341322240> **T90 Weapons**
     
 ⬥ Note: In general dual-wield camp > 2h camp at bosses due to marginally better single target dps. Using dual-wield also makes shield, Flanking, and Lunging switches easier and more fluid to use. However, one should be using both dual-wield and 2h weapons in order to use abilities that are normally exclusive to either type. 
-
+_ _
+.

```

Comment:
dont actually need to use _ _ anymore, can just leave a blank line

Reply:
Oh word will keep in mind